client/lib/domains/suggestions/index.js

```diff
@@ -1,13 +1,26 @@
+/**
+ * Internal dependencies
+ */
+import { abtest } from 'lib/abtest';
+
 /**
  * Get the suggestions vendor
  *
  * @param {boolean} [isSignup=false] Whether the query is part of a signup flow.
+ * @param {string|null} [countryCodeForABTests=null] The country code to be used for A/B tests.
  *
- * @returns {string} Vendor string to pass as part of the query.
+ * @returns {string} Vendor string to pass as part of the domain suggestions query.
  */
-export const getSuggestionsVendor = ( isSignup = false ) => {
+export const getSuggestionsVendor = ( isSignup = false, countryCodeForABTests = null ) => {
```

Comment:
Just to be clear - we're excluding Gutenboarding from this test, right?

Also cc @simison @sirreal as this might try to pull in more dependencies into Gutenberg via the abtest lib (because of https://github.com/Automattic/wp-calypso/pull/41177)

Reply:
Yes, we are excluding Gutenboarding here -- if no country code is supplied, we skip the test, but this does pull in the abtest library. :-/